var/spack/repos/builtin/packages/xeus/package.py

```diff
@@ -13,31 +15,53 @@ class Xeus(CMakePackage):
     url      = "https://github.com/QuantStack/xeus/archive/0.14.1.tar.gz"
     git      = "https://github.com/QuantStack/xeus.git"
 
+    maintainers = ['tomstitt']
+
     version('develop', branch='master')
+    version('1.0.4', sha256='7324ff013eb97d579fd3b6f9770a13f8863d6046c8bbcdbe2fc7d2ac02f0161f')
     version('0.15.0', sha256='bc99235b24d5757dc129f3ed531501fb0d0667913927ed39ee24281952649183')
     version('0.14.1', sha256='a6815845d4522ec279f142d3b4e92ef52cd80847b512146a65f256a77e058cfe')
 
     variant('examples', default=False, description="Build examples")
+    variant('shared', default=True, description="Build shared libraries")
+    variant('tests', default=False, description="Build tests")
```

Comment:
Does this change anything in the binaries being produced or installed? If not I'd avoid a variant and rely only on `self.run_tests`

Reply:
Yea, when tests are on there are some additional binaries being built

Reply:
Are those binaries installed too?

Reply:
No they are not installed as far as I can tell. How does `self.run_tests` work (is it just a bool)? Something like:

```
...
args.append(self.define('XEUS_BUILD_TESTS', self.enable_tests))
...
```

?

Reply:
Yes, something like:
```python
args.append(self.define('XEUS_BUILD_TESTS', self.run_tests))
```
should do it.